pkg/gofr/datasource/file/ftp/fs.go

```diff
@@ -84,6 +84,8 @@ func (f *fileSystem) Connect() {
 
 	ftpServer := fmt.Sprintf("%v:%v", f.config.Host, f.config.Port)
 
+	f.logger.Infof("Connecting to FTP server at '%v'", ftpServer)
```

Comment:
We should start with a small-case character for logging here just like we did for other datasources